ui/imports/shared/stores/TransactionStore.qml

```diff
@@ -170,4 +170,21 @@ QtObject {
             return qsTr("> 5 minutes")
         }
     }
+
+    property var lockedInAmounts: []
+
+    function addLockedInAmount(chainID, value, decimals, locked) {
+        let amount = Number.fromLocaleString(Qt.locale(), value) * Math.pow(10, decimals)
+        let index  = store.lockedInAmounts.findIndex(lockedItem => lockedItem !== undefined && lockedItem.chainID === chainID)
```

Comment:
I'm wondering in which case `lockedItem` might be undefined.

Reply:
it wont be, it just wont iterated if there are no items in the map. so that shouldnt occur